### DISCUSSION TITLE
Fields: Needs explanation

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#38559] [Q&A] [answered]
Working on https://docs.joomla.org/Help4.x:Fields:_Edit#Options can't find explanations for:

- Label Class (Form).
- Label Class (Output).
- Prefix.
- Suffix.
- Layout.
- Display When Read-Only.

Can someone tell me where to look?

**Comment by brianteeman** [comment, marked as answer]
- css class to apply to the field label when it is in edit mode (entering input into a field)
- css class to apply to the field label when it is displayed (displaying the output of a field)
- Fixed text to be displayed before a field eg £
- Fixed text to be displayed after a field eg EUR
- If there is a custom layout then it woould be selected here
- If the field is read only (perhaps the user doesn't have the access level) should the field be displayed or hidden

**Reply by ghost** [reply]
Thanks @brianteeman, done.
